src/Agent.Sdk/SecretMasking/ISecretMasker.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System;
+using ValueEncoder = Microsoft.TeamFoundation.DistributedTask.Logging.ValueEncoder;
+using ISecretMaskerVSO = Microsoft.TeamFoundation.DistributedTask.Logging.ISecretMasker;
+
+namespace Agent.Sdk.SecretMasking;
+public interface ISecretMasker : ISecretMaskerVSO
+{
+    new void AddRegex(String pattern);
```

Comment:
>pattern [](http://example.com/codeflow?start=29&length=7)

are you able to provider a default argument here?

`new void AddRegex(String pattern, RegexOptions regexOptions = RegexOptions.None);`

Reply:
@michaelcfanning, thanks, I added method overload